packages/ember-metal/lib/chains.js

```diff
@@ -209,11 +209,8 @@ class ChainNode {
   remove(path) {
     let paths = this._paths;
     if (paths === undefined) { return; }
-
     if (paths[path] > 0) {
       paths[path]--;
-    } else {
-      return;
```

Comment:
why not to add `assert` here to know that there was unbalanced unchain or is it too much ? @krisselden